std/js/lib/webassembly/Global.hx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C)2005-2019 Haxe Foundation
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+package js.lib.webassembly;
```

Comment:
I wonder if webassembly stuff should be in just `js.webassembly`? @haxiomic any opinion?

Reply:
I don't have a strong opinion about this one, although leaning to keeping in `lib` for consistency (the implication being `js.lib` = top-level native js externs and `js.*` = haxe-specific types that make working with the js target easier)

Reply:
My line of thought is that `js.lib` contains standard APIs that are available in any JS engine, while platform-specific APIs reside in `js.*` sub-packages, just like `js.html` and `js.node`, that's why I thought it makes sense to also have `js.webassembly`.

Reply:
I think WebAssembly (despite the name) _is_ supposed to be a standard JS engine API (it's available in v8 at least)
https://github.com/v8/v8/blob/master/src/wasm/wasm-js.cc#L2095
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly#Browser_compatibility

Reply:
WebAssembly is available in ChakraCore and node-chakuracore, too.
I think `js.lib` is better. (I think `js.webassembly` is also right.)